### PR TITLE
Crex24 - cancelOrder, cancelOrders, cancelAllOrders

### LINKED ISSUE #11529: crex24 - missing `cancelOrders`
API docs state, cancel orders by array of IDs is possible: https://docs.crex24.com/trade-api/v2/#order-cancellation
todo.

### PR DESCRIPTION
**Pull request by ttodua**
Implemented:
- cancelOrders
- cancelAllOrders with second endpoint
- cancelOrder correction

[fixes #11529]

### CODE REVIEW

File: js/crex24.js [hunk 1]
```diff
@@ -1109,31 +1114,68 @@ module.exports = class crex24 extends Exchange {
...
+            };
+            if (isSymbolArray) {
+                for (let i = 0; i < symbol.length; i++) {
+                    market = this.market (symbol);
+                    request['instruments'].push (market['id']);
+                }
+            } else {
+                market = this.market (symbol);
+                request['instruments'] = [ market['id'] ];
+            }
```
Comment (kroitor) [inline js/crex24.js:1168]: Let's not expect an array of symbols in the `symbol` argument. We don't need `isSymbolArray` here.

Comment (ttodua) [reply js/crex24.js:1168]: In docs, it states they accept multiple symbols array (`string[]`) : https://docs.crex24.com/trade-api/v2/#cancellation-by-instrument
So, I assumed we could leave users that functionality to be used, and let them pass array if they want. No?

Comment (kroitor) [reply js/crex24.js:1168]: @ttodua i'm afraid, that would lead to false-expectations from the unified api with the other exchanges.

Comment (kroitor) [reply js/crex24.js:1168]: On the other hand, we could keep it here, just for their convenience.

File: js/crex24.js [hunk 2]
```diff
@@ -1109,31 +1114,68 @@ module.exports = class crex24 extends Exchange {
...
+            //
+        } else {
+            await this.loadMarkets ();
+            const request = {
+                'instruments': undefined,
+            };
+            if (isSymbolArray) {
+                for (let i = 0; i < symbol.length; i++) {
+                    market = this.market (symbol);
+                    request['instruments'].push (market['id']);
```
Comment (kroitor) [inline js/crex24.js:1163]: Cannot push to `undefined` here.

Comment (ttodua) [reply js/crex24.js:1163]: ah, nice catch, I missed that scenario. fixed

**Pull request merged**